code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1001,6 +1001,12 @@
 			"<span class='userdanger'>[user] has attempted to [atk_verb] [target]!</span>", null, COMBAT_MESSAGE_RANGE)
 			return 0
 
+		var/teethhitcheck = rand(0,9)
```

Comment:
I'd suggest putting this onto the species themselves, so it can be changed to make specific species have strong teeth